src/test/java/org/minions/devfund/lourdes/MovieTest.java

```diff
@@ -0,0 +1,37 @@
+package org.minions.devfund.lourdes;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Implemented tests for {@link Movie}.
+ */
+public class MovieTest {
+    /**
+     * Verify Movie constructor.
+     */
+    @Test
+    public void testMovieConstructor() {
+        final String movieName = "Iron Man I";
+        Movie movie = new Movie(movieName);
+        assertEquals(movie.getName(), movieName);
+    }
+
+    /**
+     * Verify Add actor to a movie.
+     */
+    @Test
+    public void addActorOne() {
+        final String actorName = "Robert Downey";
+        final String movieName = "Iron Man I";
+        Actor actor = new Actor(actorName);
+        final ArrayList<Actor> actors = new ArrayList<Actor>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)